applications/StructuralMechanicsApplication/custom_elements/base_solid_element.h

```diff
@@ -1053,6 +1053,9 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) BaseSolidElement
             // Compute material reponse
             this->SetConstitutiveVariables(this_kinematic_variables, this_constitutive_variables, Values, point_number, integration_points);
 
+            // rotate to local axes strain/F
```

Comment:
```suggestion
            // Rotate to local axes strain/F
```

Should we check first that the element is rotated to avoid overhead?

Reply:
At that time we did it like this to try to encapsulate... I can try to separate the check

Reply:
Ideally this check should be done once per simulation but IDK when to do it... In the Initialize we may have the problem that the element has not been assigned the rotation.

Reply:
This is important, otherwise could have a significative impact